cypress/e2e/undo-redo/undo-redo.cy.js

```diff
@@ -0,0 +1,27 @@
+describe('undo-redo', () => {
+  beforeEach(() => {
+    cy.createFlow({
+      label: 'undo-redo test',
+    })
+  })
+
+  it('undo block creation', () => {
+    cy.addBlock(['Content', 'Message']).then((uuid) => {
+      cy.wait(1000)
```

Comment:
I'd suggest we create a custom cmd called `safeWait()` to encapsulate the `cy.wait(1000)`, and later we can easily change the time if needed

Reply:
Another alternative: We can put the `wait()` within undo and redo commands themselves, what do you think?